taxcalc/utils.py

```diff
@@ -681,8 +681,7 @@ def pdf_recid(recid):
     fstring = '{:' + str(col_size) + '}'
     out = out.applymap(fstring.format)
     # write ascii output to specified ascii_filename
-    out.to_csv(ascii_filename, header=False, index=False,
-               delim_whitespace=True, sep='\t')
+    out.to_csv(ascii_filename, header=False, index=False, sep='\t')
```

Comment:
yep. Bitten by the pandas philosophy "hmm, don't know that keyword arg, it's probably fine." :sad: